### PR TITLE
changes from code review

### PR DESCRIPTION
**Pull request by mmattikalli**
all html + css

### CODE REVIEW

**Comment by mmattikalli** [comment]
Closes #101

File: js/content.js [hunk 1]
```diff
@@ -237,6 +241,14 @@ function removeBlur() {
     });
 }
 
+// timer for fading in and out (works with both)
+// changing it changes the amount of time it takes to fade out (in ms)
+let timerForFading = 50;
```
Comment (Kaboomboom3) [inline js/content.js]: this should be a constant
`const FADE_LOOP_TIME` or something

File: js/content.js [hunk 2]
```diff
@@ -249,7 +261,7 @@ function fadeIn(element) {
             element.style.opacity = op;
             element.style.filter = "alpha(opacity=" + op * 100 + ")";
             op += 0.1;
```
Comment (Kaboomboom3) [inline js/content.js]: This could be a constant

File: js/content.js [hunk 3]
```diff
@@ -266,7 +278,7 @@ function fadeOut(element) {
             element.style.filter = 'alpha(opacity=' + op * 100 + ")";
             // slowly reduces opacity
             op -= 0.1;
```
Comment (Kaboomboom3) [inline js/content.js]: This could be a constant

File: js/content.js [hunk 4]
```diff
@@ -259,9 +265,10 @@ function fadeIn(element) {
                 resolve();
             }
             element.style.opacity = op;
-            element.style.filter = "alpha(opacity=" + op * 100 + ")";
-            op += 0.1;
-        }, timerForFading);
+            element.style.filter = "alpha(opacity=" + op * FADE_PERCENT + ")";
```
Comment (Kaboomboom3) [inline js/content.js]: Instead of incrementing by 0.1 and then multiplying by 100 can you just multiply by 10?

File: js/content.js [hunk 5]
```diff
@@ -243,7 +243,13 @@ function removeBlur() {
 
 // timer for fading in and out (works with both)
 // changing it changes the amount of time it takes to fade out (in ms)
-let timerForFading = 50; 
+const TIMER_FADE_LOOP = 50;
```
Comment (Kaboomboom3) [inline js/content.js]: Can this be prefixed with FADE_ instead of TIMER_ to match the rest of the constants?

File: js/content.js [hunk 6]
```diff
@@ -243,7 +243,13 @@ function removeBlur() {
...
 // timer for fading in and out (works with both)
 // changing it changes the amount of time it takes to fade out (in ms)
-let timerForFading = 50; 
+const TIMER_FADE_LOOP = 50; 
+// creates variable for how much the image is faded in/out to
+// is 100% since image should disappear or appear completely
+const FADE_PERCENT = 100; 
+// changes the "smoothness" of the fade 
+// smaller means fades in smaller intervals
+const FADE_INTERVAL = 0.1;
```
Comment (Kaboomboom3) [inline js/content.js]: Can the constants be moved to the top of the file with the rest of them? It makes it easier to find them.

File: js/content.js (hunk 4 above)
Comment (mmattikalli) [reply js/content.js]: Will not work, element.style.opacity needs a number between 0 and 1

**Pull request merged**